packages/cloud_firestore/cloud_firestore/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [UNPUBLISHED]
 
 - **FIX**: Added `==` operator override to `CollectionReferencePlatform`.
+- **FIX**: Allow iOS and macOS to be imported as a module.
```

Comment:
Not sure if I did this part right.

Reply:
This is correct, thanks for updating. We're in the middle of updating contributors guide on a separate PR 🙈